traitsui/qt4/ui_panel.py

```diff
@@ -1283,7 +1283,7 @@ class HTMLHelpWindow(QtGui.QDialog):
     """ Window for displaying Traits-based help text with HTML formatting.
     """
 
-    def __init__(self, parent, html, scale_dx, scale_dy):
+    def __init__(self, parent, html_content, scale_dx, scale_dy):
```

Comment:
Let's undo this change - it is unnecessary and it breaks backwards compatibility in a public object.

IIUC, for the scope of this method, `html` variable name will be bound to the argument passed in.

Reply:
note that the variables created in `show_help` and `show_help_popup` can still stay as `html_content` - that definitely makes the code more readable.